datafusion/core/tests/sqllogictests/test_files/window.slt

```diff
@@ -714,15 +714,27 @@ LIMIT 5
 26861 3
 
 
-#fn window_frame_ranges_preceding_following
-statement error DataFusion error: Internal error: Operator \- is not implemented for types
+# fn window_frame_ranges_preceding_following
+# when value is outside type range (10000 is outside range of tiny int (type of c2)),
+# we should treat values as infinite, hence
+# "SUM(c3) OVER(ORDER BY c2 RANGE BETWEEN 10000 PRECEDING AND 10000 FOLLOWING),"
+# is functionally equivalent to
+# "SUM(c3) OVER(ORDER BY c2 RANGE BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING),"
```

Comment:
This test supposed to run correctly, I have updated comment to explain how this query supposed to work.